autoreject/__init__.py

```diff
@@ -1,7 +1,7 @@
 __version__ = '0.2.dev0'
 
 from .autoreject import _GlobalAutoReject, _AutoReject, AutoReject
-from .autoreject import RejectLog
+from .autoreject import RejectLog, read_autoreject
```

Comment:
Shall we call it `read_reject' @jasmainak @agramfort ?

Reply:
okay for me. should I change it?